components/data-feed-service/habitat/default.toml

```diff
@@ -1,7 +1,7 @@
 [service]
 host = "localhost"
 port = 14001
-feed_interval = "30m"
+feed_interval = "1m"
```

Comment:
Any reason for the 1m change? 

I have been running with 3m for development and test purposes but. 30m seems more suitable for a production setting.

Reply:
Yeah, this just snuck in from development